initramfs/scripts/ubuntu-core-rootfs

```diff
@@ -188,15 +188,7 @@ mount_snaps()
 
 open_data_partition()
 {
-        # don't remove this sleep
-        sleep 1
-        modprobe dm-crypt
```

Comment:
Just curious: why is this not needed anymore? Will he kernel DRTR and auto-load the modules by itself?

Reply:
The kernel should load the required modules, now that they're all added to initramfs. The manual modprobes were originally added in the hope that it would provide a better error message in case of missing modules. It also seemed, in a limited number of runs, that they prevented occasional hangs in cryptsetup, but further investigation showed that the actual blocking happened in wait-for-root (meaning that's likely udev-related).